### PR TITLE
zmi: fixed Products.SiteErrorLog/issues/26

### PR DESCRIPTION
**Pull request by drfho**
Hi @icemac / @jugmac00,
here come style fix according to https://github.com/zopefoundation/Products.SiteErrorLog/issues/26
Best regards
f

fixes zopefoundation/Products.SiteErrorLog#26

### CODE REVIEW

**Comment by jugmac00** [comment]
I can confirm this fixes the problem!

Thanks a lot for taking your time!

I learned something new! Haven't heard of `inherit` before. That is exactly the reason I asked for your help instead of doing "things" :-)

**Review by jugmac00 — APPROVED** [review]

File: src/zmi/styles/resources/zmi_base.css [hunk 1]
```diff
@@ -854,6 +854,13 @@ header.navbar ul.navbar-nav li.zmi-addItemSelect select.form-control-sm:not([dis
 	margin: 1rem 0;
 	color: white;
 }
+.zmi-Site-Error-Log div.zmi-traceback.code pre {
+	margin:inherit;
+	color:inherit;
+	padding:inherit;
+	font-size:inherit;
+	overflow:inherit
```
Comment (icemac) [inline src/zmi/styles/resources/zmi_base.css:862]:
```suggestion
	overflow:inherit;
```

Is this actually necessary?

**Review by icemac — APPROVED** [review]
LGTM besides one question.

Maybe a change log entry could be nice, too.

**Comment by drfho** [comment]
> Maybe a change log entry could be nice, too.

good idea, done: https://github.com/zopefoundation/Zope/commit/eed1fce886f43bd584cfbaa6fc1c81eebd42c603

**Comment by icemac** [comment]
Thank you for fixing this issue.

**Comment by dataflake** [comment]
Please backport to `4.x` as well.

**Comment by icemac** [comment]
> Please backport to `4.x` as well.

Done in c1b899c7193ce70e6471fa347ab57df316a1446d.

**Pull request merged**